### PR TITLE
CombatFlite XML import & AH-64D elevation fix

### LINKED ISSUE #48: CombatFlite Import missing for F16/F18
#45 Pull appears to not include the Import option for the F16 or F18 in UI, and hasn't brought the Configuration changes to import CF to the F16. @PieterL1976

### LINKED ISSUE #50: Request - AH64 waypoint to not require waypoint elevation
Request to be able to leave the elevation field blank for waypoints and allow the apache to use the inbuilt terrain map. @PieterL1976

### PR DESCRIPTION
**Pull request by PieterL1976**
Closes #50 , Closes #48
- implemented Combatflite XML import for Viper & Hornet
- fixed locale bug with CombatFlite XML import
- enabled empty elevation for AH-64D waypoints

### CODE REVIEW

File: dcs-dtc/Models/AH64/Waypoints/Waypoint.cs [hunk 1]
```diff
@@ -28,8 +28,18 @@ public Waypoint(int seq, string type, string ident, string free, string mgrs,int
         public static Waypoint FromStrings(string type, string ident, string free, string mgrs, string elevation)
         {
             var match = MGRSRegex.Match(mgrs);
-            var wpt = new Waypoint(0, type, ident, free, mgrs, int.Parse(elevation));
-            return wpt;
+            try
```
Comment (p-louis) [inline dcs-dtc/Models/AH64/Waypoints/Waypoint.cs:31]: This block could be simplified to
```
int? elev = null;
int.TryParse(elevation, elev)
return new Waypoint(0, type, ident, free, mgrs, elev);
```

Also please remove the commented out lines at the end

File: dcs-dtc/Models/FA18/FA18Configuration.cs [hunk 2]
```diff
@@ -129,15 +129,9 @@ internal static FA18Configuration FromCombatFliteXML(FA18Configuration previous,
...
-                var dLat = double.Parse(lat.Replace('.', ','));
-                var dLon = double.Parse(lon.Replace('.', ','));
-                //if (
-                //    !double.TryParse(lat, out var dLat) ||
-                //    !double.TryParse(lon, out var dLon))
-                //{
-                //    continue;
-                //}
-                float.TryParse(pos.Element("Altitude")?.Value.Replace('.', ','), out var elevation);
+                var dLat = double.Parse(lat.Replace(',', '.'));
```
Comment (p-louis) [inline dcs-dtc/Models/FA18/FA18Configuration.cs:132]: Has this be verified with different locales? For example in german locale, the decimal point is replaced with a decimal comma.

**Review by p-louis — CHANGES_REQUESTED** [review]
Overall LGTM just a small change and please make sure the `replace(',','.')` stuff does work with other locales just as well.

**Comment by the-paid-actor** [comment]
Check V7 published today

**Pull request closed without merging**